canonical-websites/www.ubuntu.com
# update to japanses UA service description

## Done

Updated the Japanese service description and tided up the HTML

## QA

Check the page `/legal/ubuntu-advantage/service-description-ja` against the [updated copy](https://drive.google.com/drive/folders/0Bw_yUXzV8CSHUFA2eWNvSGVlRTQ)
